data/data/azure/vnet/nsg.tf

```diff
@@ -0,0 +1,69 @@
+resource "azurerm_network_security_group" "master" {
```

Comment:
ques: why mismatch defining rule in group and separate resources `azurerm_network_security_rule` ?

Reply:
addressed